tutorials/common/common_utils.cpp

```diff
@@ -173,12 +173,12 @@ mfxStatus LoadRawFrame(mfxFrameSurface1* pSurface, FILE* fSource)
     if (w > 2048)
         return MFX_ERR_UNSUPPORTED;
 
-    // load U
-    sts = ReadPlaneData(w, h, buf, ptr, pitch, 0, fSource);
+    // load V
+    sts = ReadPlaneData(w, h, buf, ptr, pitch, 1, fSource);
```

Comment:
Why do we need to have offset 1 here but 0 for U?

Reply:
According to [common_utils.h#L93](https://github.com/Intel-Media-SDK/MediaSDK/blob/master/tutorials/common/common_utils.h#L93), LoadRawFrame read YV12 format into NV12.
YV12 = YYYYYYYYVVUU  ,  NV12 = YYYYYYYYUVUV
That means that read YV12 with U first to NV12 is wrong in YV12 !

Reply:
Ok. If we read from plane format to NV12.